tracer/src/Datadog.Trace/LibDatadog/NativeInterop.cs

```diff
@@ -0,0 +1,67 @@
+// <copyright file="NativeInterop.cs" company="Datadog">
+// Unless explicitly stated otherwise all files in this repository are licensed under the Apache 2 License.
+// This product includes software developed at Datadog (https://www.datadoghq.com/). Copyright 2017 Datadog, Inc.
+// </copyright>
+
+#nullable enable
+
+using System;
+using System.Runtime.InteropServices;
+
+namespace Datadog.Trace.LibDatadog;
+
+internal class NativeInterop
+{
+    private const string DllName = "LibDatadog";
+
+    internal static class Exporter
+    {
+        // [DllImport(DllName, EntryPoint = "ddog_trace_exporter_new")]
+        // internal static extern ErrorHandle Create(out IntPtr outHandle, SafeHandle config);
```

Comment:
I assume these are intentionally comment out, as they'll be used shortly by the trace exporter pipeline?

Reply:
exactly